docs/stripe-setup.md

```diff
@@ -16,7 +16,7 @@ stripe listen
 
 Add the secret key to to your .env(.local):
 
-`PUBLIC_STRIPE_ENDPOINT_SECRET`
+`STRIPE_ENDPOINT_SECRET`
```

Comment:
Let's add the `VITE_PRODUCT_ID_EXAMPLEPRODUCT` and the concept of purchasable products

Reply:
Updated :)